### PR TITLE
add 387. First Unique Character in a String

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 5m
Please check my code.

### CODE REVIEW

File: 387_first-unique-character-in-a-string.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# @lc code=start
+class Solution:
+    def firstUniqChar(self, s: str) -> int:
+        """
+        time: O(n)
+        space: O(1)
+        n : len(s)
+        """
+        s_dict = collections.Counter(s)
```
Comment (ryuji0123) [inline 387_first-unique-character-in-a-string.py:17]: s_dict is unclear in this problem

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I rename the unclear name "s_dict".  Please check my code.

File: 387_first-unique-character-in-a-string.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#
+from collections import Counter
+
+
+# @lc code=start
+class Solution:
+    def firstUniqChar(self, s: str) -> int:
+        """
+        time: O(n)
+        space: O(1)
```
Comment (tkkawa) [inline 387_first-unique-character-in-a-string.py:14]: space: O(n)

**Review by tkkawa — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed space complexity.  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**